### ISSUE TITLE
Lion NaN on optimization

### ISSUE THREAD

**Issue by redknightlois** [kozistr/pytorch_optimizer#120]
Under very degenerated conditions (the architecture where I see it happen) is not very well behaved. When it works, the results are great but I need to restart many times until it works. Haven't been able to isolate the issue but seems to me that the following lines may require a few epsilons.

```
#73  p.div_(neuron_norm(p))
...
#86  grad_normed = grad / (state['exp_avg_sq'] / bias_correction).sqrt()
....
#93  p.div_(neuron_norm(p))
```

What tips me off, is that if I do a pass (single epoch) on any other optimizer and then I hook Nero in it, there is absolutely no issue found.

**Comment by kozistr** [comment]
I agree with your idea (adding an eps) too! As your experiment, `Nero` optimizer seems harder to converge at the first epoch than others though.

Anyway, Thanks for the suggestion :)

**Comment by kozistr** [comment]
I just merged the PR. maybe, I'm going to deploy a new version (v2.5.2) with fixes (#122 + more potential bugs in pt 2).

Thanks again!
